### ISSUE TITLE
Mapping Issue with Attribution Property

### ISSUE THREAD

**Issue by crashdavis** [InstaSharp/InstaSharp#64]
In the InstaSharp Media class the Attribution property is defined as a string, but instagram is returning an object, so deserialization fails.  Can you please provide a fix for this issue.  This is occurring when retrieving the recent media for a user.

**Comment by curtismbeard** [comment]
I've also just started experiencing this but needed something until this repository gets updated.  

For now, I modified the code to make Attribution an object and added the JSONIgnore attribute.  The proper solution would be to create an Attribution model.

**Comment by crashdavis** [comment]
Curtis,
I submitted a pull request with the implemented changes. Hopefully they will merge them to master branch.
On Jul 1, 2015 4:34 PM, Curtis <notifications@github.com> wrote:I've also just started experiencing this but needed something until this repository gets updated.  

For now, I modified the code to make Attribution an object and added the JSONIgnore attribute.  The proper solution would be to create an Attribution model.
